less/home-page.less

```diff
@@ -18,15 +22,45 @@
     display: flex;
     flex-wrap: wrap;
     width: 100%;
+    
+    @media @phone{
+      flex-direction:column;
+    }
+    .sky {
+      @media @phone {
+        display:none;
+      }
+    }
+    .launch {
+      @media @phone {
+        display:none;
+      }
+    }
+    .blast {
+      @media @phone {
+        width:94%
+      }
+    }
+
 
     img {
       width: 28%;
       margin: 20px 2.5%;
       border-radius: 20px;
       height: 100%;
-    }
-  }
+     }
+     .space-Images.launch{
+       @media @phone{
+         display:none
+       }
+     }
+
 
+     img{
+       width:100%
+     }
```

Comment:
You seem to declare `img { width: 100% }` a couple times. 
It might be beneficial to declare it on the global scope so you would not have to repeat yourself.